.github/workflows/latest_dependency_checker.yml

```diff
@@ -21,14 +21,15 @@ jobs:
       run: |
         python -m virtualenv venv_core
         source venv_core/bin/activate
-        make installdeps
+        python -m pip install ".[test]"
         make checkdeps OUTPUT_FILEPATH=nlp_primitives/tests/requirement_files/latest_core_dependencies.txt
         cat nlp_primitives/tests/requirement_files/latest_core_dependencies.txt
     - name: Update latest complete dependencies
       run: |
         python -m virtualenv venv_complete
         source venv_complete/bin/activate
-        make installdeps-test
+        python -m pip install ".[test]"
```

Comment:
I think we can remove this install command and just rely on `python -m pip install ".[complete]".

Reply:
no, similar to Featuretools, `complete` just installs all add-ons
- https://github.com/alteryx/nlp_primitives/blob/remove_py/setup.cfg#L59

Reply:
I thought `python -m pip install ".[complete]"` installed the base library plus all the add-ons, and we shouldn't need any of the test dependencies for checking the complete requirements.

Reply:
Fixed:
https://github.com/alteryx/nlp_primitives/pull/132/commits/b07b28395842a85243585094e58db41917f39ff7